.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Bug report
+about: Create a bug report for the project Amazon FreeRTOS
+labels: bug
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**System Information**
+- Hardware board used
+- Operating System [Windows|Linux|MacOS]
+- Version of Amazon FreeRTOS (run ``git describe --tags`` to find it)
+- Demo ran [MQTT|Shadow|Greengrass|OTA|Device Defender|TCP Echo|Sub pub Demo| Simple TCP Server]
```

Comment:
We still want to hear about Amazon FreeRTOS bugs involving hardware other than our qualified boards, and code other than our Demo and Test projects. In those cases, we may not be able to reproduce the problem, but let's consider what data we'd need in order to best be able to take action in those cases.

Reply:
I will add a `Custom User Application` option for the user to mention the project they are using, in which case they can add the code in the section `Code to reproduce the behavior`